### PR TITLE
Mila user model

### PR DESCRIPTION
**Pull request by mila-mamat**
username, email address, avatar url will be created based on **google login response**

available hours (eg: 10:00 to 18:00), available days(["Mondays","Tuesdays]), time zone will be based on **user onboarding input**

**calendarUrls** will be generated automatically based on user email, like:
**email:** mila.windsor@gmail.com 
**calendaryUrl:** https://calendly.com/mila-windsor/

**meetingurl:**  can be created in the frontend based on the calendarUrl
- https://calendly.com/mila-windsor/15min
- https://calendly.com/mila-windsor/30min
- https://calendly.com/mila-windsor/60min

### CODE REVIEW

**Review by ylee297 — APPROVED** [review]

File: server/models/User.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    },
+    // =====================================
+    //created by user during onboarding
+    // =====================================
+    timeZone: {
+      type: String,
+      enum: [
+        "UTC-3 ADT",
+        "UTC-4 EDT",
+        "UTC-5 CDT",
```
Comment (bonnieli) [inline server/models/User.js]: These are the timezones @ylee297 hard coded right? I don't think we need to restrict the strings here, instead just let it be any String cause I think there is hundreds of possibilities in terms of timezones if I'm not incorrect

File: server/models/User.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "UTC-6 MDT",
+        "UTC-7 PDT"
+      ],
+      required: true,
+    },
+    //not decided on available hour yet
+    availableHours: {
```
Comment (bonnieli) [inline server/models/User.js]: I'm wondering if we should do `startAvailableHour` and `endAvailableHour` instead

File: server/models/User.js (hunk 1 above)
Comment (mila-mamat) [reply server/models/User.js]: I'm wondering how the calendar would incorporate different time zones later. If we only rely on google calendar, it should be taken care of. But if we want to save the appointments in the database, and check it manually, wouldn't it be too complicated if there were hundreds of them?

Comment (mila-mamat) [reply server/models/User.js]: besides, R we stroting the timezone code ( utc-3, ADT), or the name from moment.js (America/Toronto)?

**Pull request merged**